modules/ossm-sidecar-validate-kiali.adoc

```diff
@@ -0,0 +1,31 @@
+////
+This module included in the following assemblies:
+* service_mesh/v2x/prepare-to-deploy-applications-ossm.adoc
+////
+:_content-type: CONCEPT
+[id="ossm-validating-sidecar_{context}"]
+= Validating sidecar injection
+
+The Kiali console offers several ways to validate whether or not your applications, services, and workloads have a sidecar proxy.
+
+.Missing sidecar badge
+image::ossm-node-badge-missing-sidecar.svg[Missing Sidecar badge]
+
+The *Graph* page displays a node badge indicating a *Missing Sidecar* on the following graphs:
+
+* App graph
+* Versioned app graph
+* Workload graph
+
+.Missing sidecar icon
+image::ossm-icon-missing-sidecar.png[Missing Sidecar icon]
```

Comment:
Same as above.

<img width="204" alt="image" src="https://user-images.githubusercontent.com/83772007/162497718-bca118c5-8821-4df9-ad2f-cfa264ea4d45.png">

Reply:
Yes, I'm laughing because this could be confusing to review if you don't know the GUI!